develop/smart-contracts/overview.md

```diff
@@ -18,8 +18,7 @@ This guide outlines the primary approaches to developing smart contracts in the
 
 You'll explore the key differences between these development paths, along with considerations for parachain developers integrating smart contract functionality.
 
-!!!info "Parachain Developer?"
-    If you are a parachain developer looking to add smart contract functionality to your chain, please refer to the [Add Smart Contract Functionality](/develop/parachains/customize-parachain/add-smart-contract-functionality/){target=\_blank} page, which covers both Wasm and EVM-based contract implementations.
+If you are a parachain developer looking to add smart contract functionality to your chain, please refer to the [Add Smart Contract Functionality](/develop/parachains/customize-parachain/add-smart-contract-functionality/){target=\_blank} page, which covers both Wasm and EVM-based contract implementations.
```

Comment:
Maybe an admonition is not the right thing here, but I think something to highlight this section is important

Reply:
I'm open to suggestions that aren't admonitions. Taking it out of the box makes it more likely a person will read it. We could add a subheading (h3) to make it stand out and appear in the TOC on the right. Do you have any other thoughts?